src/js/core/methods/helpers/cardanoCertificate.js

```diff
@@ -58,12 +61,12 @@ const validatePoolRelay = (relay: CardanoPoolRelay) => {
                 'Either ipv4Address or ipv6Address must be supplied',
             );
         }
-    } else if (relay.type === POOL_RELAY_TYPE.SingleHostName) {
+    } else if (relay.type === CardanoPoolRelayType.SINGLE_HOST_NAME) {
         validateParams(relay, [
             { name: 'hostName', type: 'string', obligatory: true },
             { name: 'port', type: 'number', obligatory: true },
         ]);
-    } else if (POOL_RELAY_TYPE.MultipleHostName) {
+    } else if (relay.type === CardanoPoolRelayType.MULTIPLE_HOST_NAME) {
```

Comment:
@gabrielKerekes whats your opinion on this change?
previous condition would always return true (MULTIPLE_HOST_NAME = 2)

Reply:
This definitely seems to be a bug - not a serious one, but still a bug. The change makes sense :+1: